interface/modules/custom_modules/oe-module-claimrev-connect/templates/eligibility.php

```diff
@@ -0,0 +1,255 @@
+<?php 
+/**
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ *
+ * @author    Brad Sharp <brad.sharp@claimrev.com>
+ * @copyright Copyright (c) 2022 Brad Sharp <brad.sharp@claimrev.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+use OpenEMR\Modules\ClaimRevConnector\EligibilityData;
+use OpenEMR\Modules\ClaimRevConnector\EligibilityInquiryRequest;
+use OpenEMR\Modules\ClaimRevConnector\SubscriberPatientEligibilityRequest;
+use OpenEMR\Modules\ClaimRevConnector\EligibilityObjectCreator;
+use OpenEMR\Modules\ClaimRevConnector\ValueMapping;
+
+
+if($pid == null){
+    echo xlt("Error retrieving patient.");
+    exit;
+}
+$insurance = EligibilityData::getInsuranceData($pid);
+
+if(isset($_POST['checkElig'])) { //check if form was submitted
+
+    $pr=$_POST['responsibility'];
+    //$pid is found on the parent page that is including this php file
+    $formatedPr = ValueMapping::MapPayerResponsibility($pr);
+    EligibilityData::RemoveEligibilityCheck($pid,$formatedPr);
+    $requestObjects = EligibilityObjectCreator::BuildObject($pid,$pr,null,null,null);
+    EligibilityObjectCreator::SaveToDatabase($requestObjects,$pid );
+    $request = $requestObjects[0];  
+}
+
+?>
+
+
+
+
+
+<div class="row">
+    <div class="col">
+    <ul class="nav nav-tabs mb-2">
+        <?php  $classActive = "active"; $first="true"; foreach ($insurance as $row) { ?>
+            <li class="nav-item" role="presentation">
+                <a id="claimrev-ins-<?php echo(ucfirst($row['payer_responsibility']));?>-tab" aria-selected="<?php echo($first); ?>" class="nav-link <?php echo($classActive);?>"  data-toggle="tab" role="tab" href="#<?php echo(ucfirst($row['payer_responsibility']));?>"> <?php echo xlt(ucfirst($row['payer_responsibility']));?>  </a>
+            </li>
+        <?php $first = "false"; $classActive=""; } ?>
+        
+    </ul>
+    <div class="tab-content">
+        <?php 
+        $classActive = "in active"; 
+        foreach ($insurance as $row) { 
+            $eligibilityCheck = EligibilityData::getEligibilityResult($pid,$row['payer_responsibility']);
+            ?>
+            <div id="<?php echo(ucfirst($row['payer_responsibility']));?>" class="tab-pane <?php echo($classActive);?>">
+                <div class="row">
+                    <div class="col-2">
+                    
+                        <form method="post" action="<?php echo htmlspecialchars($_SERVER["PHP_SELF"]); ?>">
+                            <input type="hidden" id="responsibility" name="responsibility" value="<?php echo(ucfirst($row['payer_responsibility']));?>">
```

Comment:
would also put attr() around the ucfirst($row['payer_responsibility']) in the value attribute